bsmetadata/hydra_configs/v2.yaml

```diff
@@ -27,8 +27,8 @@ data_config:
     metadata_sep: ' | '
     metadata_key_value_sep: ': '
     metadata_probability: 1.0
-    treat_local_metadata_as_regular_text: false
-    add_local_metadata_special_tokens_in_prefix: false
+    treat_local_metadata_as_regular_text: true
```

Comment:
If I recall correctly this argument is used to know if we want to take the local metadata occurrences into account in the loss: I think that we want them to be taken into account.

Do you agree @manandey , @cccntu ?

Reply:
Yeah, I completely agree! :)

Reply:
Yes, I think it makes more sense to calculate the loss of html, too.